changelog.txt

```diff
@@ -1,4 +1,5 @@
 V1.XX.X
+[QMS-594] Java version check fails in BRouter setup
```

Comment:
still increasing order top to bottom :wink:

Reply:
fixed :-)